test/unit/views/ChoiceViewSpec.scala

```diff
@@ -85,12 +85,12 @@ class ChoiceViewSpec extends UnitViewSpec with ChoiceMessages with CommonMessage
 
       val view = createView(Choice.form())
 
-      view.select("div.multiple-choice:nth-child(2)").text() must be(messages(arrivalDecLabel))
-      view.select("div.multiple-choice:nth-child(3)").text() must be(messages(associateDecLabel))
-      view.select("div.multiple-choice:nth-child(4)").text() must be(messages(disassociateDecLabel))
-      view.select("div.multiple-choice:nth-child(5)").text() must be(messages(shutMucrLabel))
-      view.select("div.multiple-choice:nth-child(6)").text() must be(messages(departureDecLabel))
-      view.select("div.multiple-choice:nth-child(7)").text() must be(messages(viewSubmissionsLabel))
+      view.getElementsByClass("govuk-label govuk-radios__label").get(0).text() must be(messages(arrivalDecLabel))
```

Comment:
Maybe would be worth to pick little different class - we could go via utility way like `[for='id-for-field']` or go little by BEM
```.govuk-radios__label.cds_choice__arrival``` - what i'am not like is we depend on order here.

Reply:
Good recommendation.